pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -88,20 +110,21 @@ public InputStream getStream() throws IOException {
     if (stream != null) {
       return stream;
     }
-    StreamWrapperFinalizeAction finalizeAction = this.finalizeAction;
+    LazyCleaner.CleaningAction finalizeAction = this.finalizeAction;
     if (finalizeAction != null) {
-      return finalizeAction.getStream();
+      InputStream stream = ((StreamWrapperCleaningAction) (finalizeAction)).getStream();
```

Comment:
Should StreamWrapper rather refer the stream directly?

Reply:
that would allow getting rid of holding the reference to finalizAction